### PR TITLE
Grid Component

### PR DESCRIPTION
**Pull request by sahiba0915**
Files to check:

1. grid.html
2. grid.css

Live Preview: https://gridcomp.netlify.app/component/grid/grid.html

### CODE REVIEW

File: component/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-auto-rows: minmax(100px, auto);
+  gap: 10px;
+  background-color: var(--primary-color);
+  text-align: center;
+}
+
+.grid-item {
+  height: 100px;
```
Comment (akash-1318) [inline component/grid/grid.css:26]: in case you forgot don't use pixel here instead use rem.

**Review by akash-1318 — COMMENTED** [review]
Just added a suggestion else everything looks good from my end , really clean code .

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks neat and clean

File: component/grid/grid.css (hunk 1 above)
Comment (sahiba0915) [reply component/grid/grid.css:26]: yes will do that

**Pull request merged**